setup.cfg

```diff
@@ -43,7 +43,12 @@ classifiers =
 
 [options]
 install_requires =
+    xmltodict
```

Comment:
We need version pins on all these requirements in order to avoid the [Dependency Confusion Vulnerability](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610).